hadoop-ozone/ozone-manager/src/test/java/org/apache/hadoop/ozone/web/ozShell/TestObjectPrinter.java

```diff
@@ -40,11 +40,13 @@ public void printObjectAsJson() throws IOException {
     OzoneConfiguration conf = new OzoneConfiguration();
     OzoneVolume volume =
         new OzoneVolume(conf, Mockito.mock(ClientProtocol.class), "name",
-            "admin", "owner", 1L, 0L,
+            "admin", "owner", 1L, Instant.ofEpochMilli(0).toEpochMilli(),
```

Comment:
This seems a complex way of writing `0`.  If the intention is to avoid hard-coding the value, I think this one would be simpler:

```
Instant.EPOCH.toEpochMilli()
```

Reply:
Fixed.